troubleshooting/troubleshooting-vse-path-too-long-exception.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Path Too Long Exception when Building Project in Microsoft Visual Studio
+description: AppBuilder Projects with fully qualified file names longer that 260 symbols fail to build in Visual Studio Extension.
+type: troubleshooting
+page_title: VS Extension Build Fails when File Path Length is Longer than 260 Symbols 
+slug: vse-path-too-long-exception
+position:
+tags: appbuilder, vs-extension, build
+teampulseid:
+ticketid:  1044168, 1001926
+pitsid:
+
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>	
+  </tr>
+  <tr>
+    <td>Code (AppBuilder) client</td>
+    <td>Extension for Visual Studio</td>	
+  </tr>
+</table>
+
+## Description
+When you attempt to build your AppBuilder project in Microsoft Visual Studio, the following exception occurs: `System.IO.PathTooLongException`
+
+## Error Message
+###`System.IO.PathTooLongException`
```

Comment:
Is there any particular reason for the H3 here?

Try using code block instead of inline code formatting. If you don't think it improves on the user experience, you can revert to inline code formatting.